docs/docs/fundamentals/installation.md

```diff
@@ -23,17 +23,15 @@ Note that if you wish to use [`React.createRef()`](https://reactjs.org/docs/refs
 
 In order to fully utilize the [touch events](/docs/gestures/touch-events/) you also need to use `react-native-reanimated` 2.3.0-beta.4 or newer.
 
-## Expo
+## [Expo](https://expo.dev)
 
-### Managed [Expo](https://expo.io)
+To use the version of react-native-gesture-handler that is compatible with your Expo project, run `npx expo install react-native-gesture-handler`.
 
-To use the version of react-native-gesture-handler that is compatible with your managed Expo project, run `expo install react-native-gesture-handler`.
+The Expo SDK incorporates the latest version of react-native-gesture-handler available at the time of each SDK release, so Expo apps might not always support all our latest features as soon as they are available.
 
-The Expo SDK incorporates the latest version of react-native-gesture-handler available at the time of each SDK release, so managed Expo apps might not always support all our latest features as soon as they are available.
+### Bare [React Native](https://reactnative.dev/)
 
-### Bare [React Native](http://reactnative.dev/)
-
-Since the library uses native support for handling gestures, it requires an extended installation to the norm. If you are starting a new project, you may want to initialize it with [expo-cli](https://docs.expo.io/versions/latest/workflow/expo-cli/) and use a bare template, they come pre-installed with react-native-gesture-handler.
+Since the library uses native support for handling gestures, it requires an extended installation to the norm. If you are starting a new project, initialize it with [`npx create-expo-app@latest`](https://docs.expo.dev/more/create-expo/) since it comes pre-installed with react-native-gesture-handler.
```

Comment:
```suggestion
Since the library uses native support for handling gestures, it requires an extended installation to the norm. If you are starting a new project, initialize it with [`npx create-expo-app@latest`](https://docs.expo.dev/get-started/create-a-project/) since it comes pre-installed with react-native-gesture-handler.
```